src/type-coercions.md

```diff
@@ -159,7 +159,7 @@ Coercion is allowed between the following types:
 ### Unsized Coercions
 
 The following coercions are called `unsized coercions`, since they
-relate to converting sized types to unsized types, and are permitted in a few
+relate to converting types to unsized types, and are permitted in a few
 cases where other coercions are not, as described above. They can still happen
 anywhere else a coercion can occur.
```

Comment:
IMO it is worth stating explicitly that "unsizing" is a misnomer now and this also covers coercions from unsized types to unsized types.